tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py

```diff
@@ -0,0 +1,102 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import Base
+
+
+class PlayerView(Base):
+    _audio_locator = (By.CSS_SELECTOR, '#audio')
+    _active_view_locator = (By.CSS_SELECTOR, 'iframe.active[src*="/views/player/index.html"]')
+    _player_seek_elapsed_locator = (By.ID, 'elapsed-time')
+    _player_controls_shadow_DOM_locator = (By.ID, 'controls')
+    _player_controls_play_locator = (By.ID, 'toggle')
+    _cover_image_shadow_DOM_locator = (By.ID, 'artwork')
+    _seek_bar_shadow_DOM_locator = (By.ID, 'seek-bar')
+    _cover_share_locator = (By.CSS_SELECTOR, '[data-l10n-id="share-song"]')
+    _rating_view_locator = (By.CSS_SELECTOR, '#rating')
```

Comment:
actually, strangely enough, when I used 'By.ID', it didn't find the element.  I think it's either some kind of marionette bug or shadow dom issue.